lib/util.py

```diff
@@ -513,7 +515,7 @@ def block_explorer_info():
     return testnet_block_explorers if constants.net.TESTNET else mainnet_block_explorers
 
 def block_explorer(config):
-    return config.get('block_explorer', 'fsight.chain.tips')
+    return config.get('block_explorer', 'fsight.chain.tips', 'chainz.cryptoid.info')
```

Comment:
This doesn't seem to be needed. When I apply your changes and click a bit in the menu I get the following error in the console
> Traceback (most recent call last):
>  File "/home/filbert/src/electrum-ftc/gui/qt/main_window.py", line 2761, in settings_dialog
>    block_ex_combo.setCurrentIndex(block_ex_combo.findText(util.block_explorer(self.config)))
>  File "/home/filbert/src/electrum-ftc/lib/util.py", line 518, in block_explorer
>    return config.get('block_explorer', 'fsight.chain.tips', 'chainz.cryptoid.info')
>TypeError: get() takes from 2 to 3 positional arguments but 4 were given

The error message is a bit confusing because Python counts `config.` as the first argument.

Reply:
It therefore would work without being applied there?

Reply:
Yes, if you check the [implementation of `config.get()`](https://github.com/Feathercoin-Foundation/electrum-ftc/blob/5f72cadd777d80a7235e4860589c425287a67fe9/lib/simple_config.py#L155) you can see that the last argument it takes is just the default value in case the user hasn't set a config value for `'block_explorer'`. That's also why `'explorer.feathercoin.com'` from the list above doesn't appear here as well.

Reply:
Aha, okay - thank you! Ì took it out now.
Means fsight.chain.tips is mentioned there to be the default block explorer? (sorry, just to be sure I understood it fully).

Reply:
Exactly.